Cargo.toml

```diff
@@ -38,28 +38,27 @@ chrono = { version = "0.4", optional = true }
 
 # Parity
 primitive-types = "0.7"
-jsonrpsee = { git = "https://github.com/dt665m/jsonrpsee", branch = "feature/client-error-handling" }
 kvdb = "0.7"
 kvdb-rocksdb = "0.9"
 codec = { package = "parity-scale-codec", version = "1.3", default-features = false, features = ["derive", "full"] }
 hash-db = "0.15"
 
 # Substrate
-sp-database = { git = "https://github.com/paritytech/substrate", branch = "master", package="sp-database" }
-sp-storage = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-storage" }
-sp-externalities = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-externalities" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-runtime" }
-sc-client-api = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sc-client-api" }
-sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-blockchain" }
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-core" }
-sp-api = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-api" }
-sp-block-builder = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-block-builder" }
-sp-version = {  git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-version" }
-sc-executor = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sc-executor" }
-sc-chain-spec = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sc-chain-spec" }
-sp-trie = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-trie" }
-sp-state-machine = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-state-machine" }
-sp-io = { git = "https://github.com/paritytech/substrate", branch = "master", package = "sp-io" }
+sp-database = { git = "https://github.com/paritytech/substrate", package="sp-database" , rev = "647ad15565d7c35ecf00b73b12cccad9858780b9" }
```

Comment:
Outside of the scope of this PR, any idea i the new substrate versioning allow us to use `tag` instead of `rev` in the future?

Reply:
I hope so, or even better, just straight up crate versions. :)

I believe this PR is rather temporary though, as of the next kusama/polkadot releases we should be able to use the `2.0.0` releases.